qml/MainContent.qml

```diff
@@ -35,10 +35,15 @@ Item {
         Decorative {}
     }
 
+    MouseArea {
+        anchors.fill: parent
+        onClicked: forceActiveFocus()
```

Comment:
Why you need it? I'm just curious

Reply:
This is to be able cancel editing (for example time or cycle) just by clicking outside of the widget. This way when a background is clicked it gathers the focus so the other widget looses it.